gwt-regexp/pom.xml

```diff
@@ -63,12 +63,6 @@
             <artifactId>elemental2-core</artifactId>
             <version>${elemental2.version}</version>
         </dependency>
-        <dependency>
-            <groupId>com.google.elemental2</groupId>
-            <artifactId>elemental2-core</artifactId>
-            <version>${elemental2.version}</version>
-            <classifier>sources</classifier>
```

Comment:
All elemental2 jars come with their own sources already - declaring another copy makes j2cl build both copies separately, and try to combine them in closure, which can cause compile errors (it can't tell they are the same).